Applications/MainApplication/Gui/MaterialEditor.cpp

```diff
@@ -79,8 +81,8 @@ void MaterialEditor::newKdColor( const QColor& color ) {
 
     if ( m_renderObject && m_usable )
     {
-        Core::Utils::Color c( color.redF(), color.greenF(), color.blueF(), 1.0 );
-        m_material->m_kd = c;
+        m_blinnphongmaterial->m_kd = Core::Utils::Color (
+              color.redF(), color.greenF(), color.blueF(), 1. );
```

Comment:
`1_ra` ?

Reply:
No, `blueF` and others are double, so we need a double here to have all the parameters with the same type (see `ColorBase` constructor)

Reply:
Color works from `Scalar`s, so these should all be as such right?

Reply:
There is a constructor with a template parameter allowing to create a `ColorBase<Scalar>` from 4 values of another Scalar type, that are altogether converted to `Scalar` in the constructor.
As such, the 4 types given as input needs to be of the same type, otherwise there is a confusion for the compiler. This is done in purpose by design, to force the user to give consistent type and avoid unwanted implicit conversions.

Makes sense ?

Reply:
ok